scoresheets/Restaurant.tex

```diff
@@ -18,9 +18,9 @@
 
 	\scoreheading{Deus ex Machina Penalties}
 	\penaltyitem[2]{200}{Being guided to a table.}
-	\penaltyitem[6]{50}{Asking Barman to handover object.}
-	\penaltyitem[6]{50}{Handing over the object to the guest.}
-	\penaltyitem[2]{100}{Being told/pointed where is a table/\textit{Kitchen-bar} (per order).}
+	\penaltyitem[3]{100}{Asking the Barman to handover object to the robot.}
```

Comment:
Why 3 times? As you can only take an order from a guest 2 times, correct?

Reply:
So, as it is written right now. If you get an order with 2 objects there are 2 pick and 2 place actions that can be skipped. But for an order with 3 objects there are 3 each that can be skipped. Therefore 3 penalties (or 6 combined as it was written before).
As I mentioned above I think this is bad because it makes getting an order with 3 objects even more punishing. So in my opinion for an order with 3 objects even if 3 pick and 3 place actions are skipped the penalty should only be applied twice to make it even with 2 objects orders. But I wanted some feedback for this because it seemed like a bigger change.

Reply:
With your changes, bringing an order with attached container(like Tech United did in Portugal), will result in zero points for 1 and 2 item orders. And a 3 item order would result in 200 points.

I think that is penalizing too much.

Reply:
No, how it currently is you'd get 0 points for a three item order and 200 points for a 2 item order. I would like to change it so your method would give 200 points in both cases

Reply:
You will never a 1 item order?

Reply:
Right, discrepancy is even worse then...
I already think having 2 and 3 item orders is bad because of the difference in difficulty 1 vs 3 is obviously worse.
@MatthijsBurgh Do you think it is still fine to change it to order is 2 to 3 objects and penalties can only be applied to two objects per order?

Reply:
I agree that bringing an order without manipulation should result in the same amount of points for 2 and 3 item orders.

So when bringing an order is 600. Skipping manipulating the object is 100 points per skip. And skipping can happen both at the bar and the guest table. When limiting the reduction to only to 2 objects, so 4 skip. You end with 200 points.

So I don't think it matters whether the skips are at the bar or the guest table. So just 4 skips would also be fine by me.

I also agree with you we can drop 1 item orders. Or we should change the scoring, so that manipulation grants points instead of penalizing not manipulating.